src/tools/docker-run-to-docker-compose-converter/docker-run-to-docker-compose-converter.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <n-card>
+    <n-form-item label="Your docker run command:">
+      <n-input
+        v-model:value="dockerRun"
+        style="font-family: monospace"
+        type="textarea"
+        placeholder="Your docker run command to convert..."
+        rows="3"
+      />
+    </n-form-item>
+
+    <n-divider />
+
+    <textarea-copyable :value="dockerCompose" language="yaml" />
+    <br />
+    <br />
+    <n-space justify="center">
+      <n-button :disabled="dockerCompose === ''" secondary @click="download"> Download docker-compose.yml </n-button>
+    </n-space>
+  </n-card>
```

Comment:
Maybe switch to a div instead of a card ? It adds unecessary layers

```suggestion
  <div>
    <n-form-item label="Your docker run command:">
      <n-input
        v-model:value="dockerRun"
        style="font-family: monospace"
        type="textarea"
        placeholder="Your docker run command to convert..."
        rows="3"
      />
    </n-form-item>

    <n-divider />

    <textarea-copyable :value="dockerCompose" language="yaml" />
    <br />
    <br />
    <n-space justify="center">
      <n-button :disabled="dockerCompose === ''" secondary @click="download"> Download docker-compose.yml </n-button>
    </n-space>
  </div>
```